### ISSUE TITLE
RWKV6 backward gives nan gradients

### ISSUE THREAD

**Issue by hypnopump** [sustcsonglin/flash-linear-attention#11]
U gradient is fine. all other grads grow uncontrolably

```python
import torch as th
from fla.ops.rwkv_6.recurrent_fuse import fused_recurrent_rwkv6

B, H, L, K, V = 2, 4, 256, 64, 64

r, k, v, w = th.randn(4, B, H, L, K).cuda()
w = w.sigmoid()
u = th.randn(H, K).cuda()

r.requires_grad = True
k.requires_grad = True
v.requires_grad = True
w.requires_grad = True
u.requires_grad = True

o, state = fused_recurrent_rwkv6(r, k, v, w, u)

print(o.shape)
o.mean().backward()
print(u.grad.shape)
print(w.grad)
```

**Comment by hypnopump** [comment]
solved by passing `w` already in log space as suggested; thx
